### DISCUSSION TITLE
doris使用iceberg外表无法读取hadoop集群名称

### DISCUSSION THREAD

**Discussion by lyteyungs** [apache/doris#9381] [A - General / Q&A]
异常为 java.lang.IllegalArgumentException: java.net.UnknownHostException:   ,已经将 core-site.xml  和 hdfs-site.xml  放到了 fe/conf/ 和 be/conf/ 下 ,但是还是读不到集群信息  ,这两个配置文件应该放在什么地方才能正常被读取集群信息?

**Comment by morningman** [comment]
Could you provide your create table statement?

**Reply by hf200012** [reply]
Does your HDFs have ha mode enabled?

**Reply by hf200012** [reply]
https://github.com/apache/incubator-doris/pull/9151

**Reply by lyteyungs** [reply]
![image](https://user-images.githubusercontent.com/7997076/167051230-5e86cacd-34ab-42fd-b725-98114f1501d0.png)
这是在presto里查询的效果 ,  
![image](https://user-images.githubusercontent.com/7997076/167051298-0c2d0e79-6d46-4cb8-b9eb-e02bd683f37f.png)
这是在doris里建表的效果,卡住不动, 
![image](https://user-images.githubusercontent.com/7997076/167051367-2e536518-1891-44d1-a86e-93cc8822b729.png)
这是fe报错未知 集群名称, 
![image](https://user-images.githubusercontent.com/7997076/167051506-4957fe14-6d75-4a95-9531-b3ecc939ab41.png)
这是配置文件的位置,
![image](https://user-images.githubusercontent.com/7997076/167051733-63df98db-6fa6-4813-b1f2-5cfcab5d2a7c.png)
![image](https://user-images.githubusercontent.com/7997076/167051769-07c271fc-a044-4526-a790-01c51c283ecf.png)
![image](https://user-images.githubusercontent.com/7997076/167051855-7360d9d5-5827-42bd-b69d-de634180756f.png)
配置文件中是包含这个名称的机器位置的,
![image](https://user-images.githubusercontent.com/7997076/167051901-9a6dd2a1-c203-4c85-b89b-cc99156f3efc.png)
这些指向的host 也是被配置过的 ,
集群有ha

**Reply by hf200012** [reply]
https://github.com/apache/incubator-doris/pull/9151
使用这个补丁
